Atomic/syscontainers.py

```diff
@@ -227,6 +227,121 @@ def build_rpm(self, repo, name, image, values, destination):
         return None
 
     def install(self, image, name):
+        """
+        External container install logic.
```

Comment:
I know we are not outwardly changing at of the interfaces to the users which usually prompts someone to do updates a docs/ file.  I'm thinking however it might be worth noting this change and the labels in the docs somwhere.